internal/command/project_metadata.go

```diff
@@ -0,0 +1,205 @@
+package command
+
+import (
+	"context"
+
+	"github.com/zitadel/zitadel/internal/domain"
+	"github.com/zitadel/zitadel/internal/eventstore"
+	"github.com/zitadel/zitadel/internal/repository/project"
+	"github.com/zitadel/zitadel/internal/zerrors"
+)
+
+func (c *Commands) SetProjectMetadata(ctx context.Context, projectID, resourceOwner string, metadata *domain.Metadata) (_ *domain.Metadata, err error) {
```

Comment:
Do we even need `SetProjectMetadata`?
As BulkSetProjectMetadata is only used in the API.